pom.xml

```diff
@@ -115,6 +97,53 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>dse</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <modules>
+        <module>core</module>
+        <module>stargate-starter</module>
+        <module>persistence-api</module>
+        <module>persistence-common</module>
+        <module>persistence-cassandra-3.11</module>
+        <module>persistence-cassandra-4.0</module>
+        <module>persistence-dse-6.8</module>
+        <module>cql</module>
+        <module>graphqlapi</module>
+        <module>restapi</module>
+        <module>auth-api</module>
+        <module>authentication</module>
+        <module>auth-table-based-service</module>
+        <module>health-checker</module>
+        <!-- testing last -->
+        <module>testing</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>core</module>
+        <module>stargate-starter</module>
+        <module>persistence-api</module>
+        <module>persistence-common</module>
+        <module>persistence-cassandra-3.11</module>
+        <module>persistence-cassandra-4.0</module>
+        <module>cql</module>
+        <module>graphqlapi</module>
+        <module>restapi</module>
+        <module>auth-api</module>
+        <module>authentication</module>
+        <module>auth-table-based-service</module>
+        <module>health-checker</module>
+        <!-- testing last -->
+        <module>testing</module>
+      </modules>
```

Comment:
Do we need to have default modules in a profile? Could we put them in the main (top-level) `modules` block? Something like that works with configs... I wonder if it might work with modules too. Having an unconditional section, plus the `dse` module _addition_ in a profile would be more readable I think :thinking: